catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -87,6 +89,8 @@
 
   private EndpointUtil util;
 
+  private MessageValidatorManager messageValidatorManager;
```

Comment:
✏️ If we use constructor injection, this variable can be `final`, which means it cannot be reassigned. The more invariants we can enforce, the easier it is to reason about code.